assets/src/scripts/monitoring-location/components/hydrograph/index.js

```diff
@@ -141,18 +152,28 @@ export const attachToNode = function(store,
             // Initial data has been fetched and initial state set. We can render the hydrograph elements
             // Set up rendering functions for the graph-container
             let graphContainer = nodeElem.select('.graph-container')
-                .call(link(store, controlDisplay, hasAnyTimeSeries))
-                .call(drawTimeSeriesGraph, store, siteno, showMLName, !showOnlyGraph);
-            if (!showOnlyGraph) {
-                graphContainer
-                    .call(drawTooltipCursorSlider, store)
-                    .call(drawGraphBrush, store);
-            }
-
-            graphContainer.append('div')
-                .classed('ts-legend-controls-container', true)
-                .call(drawTimeSeriesLegend, store);
+                .call(link(store, controlDisplay, hasAnyTimeSeries));
 
+            if (window.navigator.userAgent.indexOf('MSIE') !== -1) {
+                graphContainer.call(link(store, getStaticGraph, createStructuredSelector({
```

Comment:
If D3 doesn't work in IE11, this won't work either. Right now, I believe on IE11, you will not be able to change the parameterCode or anything else because all of that code is rendered using D3 and doesn't appear. 

I think we may trying to be too smart here. What if we just add a snippet of javascript to https://github.com/usgs/waterdataui/blob/master/wdfn-server/waterdata/templates/base_plain.html to detect the browser and then add the img tag with the png.

Reply:
That is an easy way to do it, and I implemented a version of that on day one. But then when I tested it, I realized that all the page function is lost and people can only get from page to page by an existing link or by direct entry of the URL. Is that an acceptable compromise? I guess, it maybe the only way to get this to work.

Reply:
But as it stands now, there is nothing on the page except the summary of all available data and location metadata. They can't change anything anyway. At least this way they get a hydrograph.